src/olympia/reviewers/models.py

```diff
@@ -1111,6 +1115,16 @@ def check_uses_native_messaging(cls, version):
         return any('nativeMessaging' in file_.webext_permissions_list
                    for file_ in version.all_files)
 
+    @classmethod
+    def check_is_listing_disabled(cls, version):
+        """Check whether the add-on is disabled or incomplete.
+
+        Only applies to listed versions."""
```

Comment:
should the `addon.is_disabled` check apply to unlisted versions too?

Reply:
See below.